src/main/java/com/softserveinc/dokazovi/mapper/PostMapper.java

```diff
@@ -0,0 +1,15 @@
+package com.softserveinc.dokazovi.mapper;
+
+import com.softserveinc.dokazovi.dto.post.LatestPostDTO;
+import com.softserveinc.dokazovi.entity.PostEntity;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(componentModel = "spring", uses = {PostUserMapper.class, PostTypeMapper.class, PostDirectionMapper.class})
+public interface PostMapper {
```

Comment:
LatestPostMapper fits better in this context.

Reply:
I think it's not a problem to map different DTOs, belonging to the same entity,  with one mapper

Reply:
Agree